telemetry/packages/public-app/app/globals.css

```diff
@@ -0,0 +1,232 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+.fade-in-image {
+  animation: fadeIn 6s;
+}
+
+@keyframes fadeIn {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+.processing {
+  color: white;
+  font-size: small;
+  margin-top: -10px;
+}
+
+.btn {
+  color: black;
+}
+
+.inverted {
+  filter: invert(100%);
+  color: black;
+}
+
+.invert-mode {
+  color: var(--invert-color);
+}
+
+.levitation {
+  transform: rotate(90deg);
+  /* margin-bottom: 100px; */
+}
+
+.levitate {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  /* align-self: center;
+  justify-self: center;
+   */
+  /* margin-bottom: 10px; */
+}
+
+.levitate-2 {
```

Comment:
What's `levitate-2`? Might need to use some more descriptive namings for the CSS variables